pkgs/development/libraries/mesa/default.nix

```diff
@@ -188,12 +198,13 @@ self = stdenv.mkDerivation {
     mkdir -p $opencl/lib
     mv -t "$opencl/lib/"     \
       $out/lib/gallium-pipe   \
-      $out/lib/libMesaOpenCL*
+      $out/lib/lib*OpenCL*
 
-    # We construct our own .icd file that contains an absolute path.
+    # We construct our own .icd files that contain absolute paths.
```

Comment:
```suggestion
    # We construct our own .icd files that contains absolute paths.
```

Reply:
That suggestion is definitely not correct English.